lib/mapObjects/CommonConstructors.h

```diff
@@ -171,6 +171,8 @@ class DLL_LINKAGE CBankInfo : public IObjectInfo
 	CBankInfo(const JsonVector & Config);
 
 	TPossibleGuards getPossibleGuards() const;
+	TResources getPossibleResourcesReward() const;
+	std::vector<CStackBasicDescriptor> getPossibleCreaturesReward() const;
```

Comment:
This is an additional API for AI to understand what object can give when visited.